ios/KalturaPlayerViewManager.swift

```diff
@@ -554,7 +554,18 @@ class KalturaPlayerRNView : UIView {
     }
     
     func updateOttAnalyticsConfig(config: Dictionary<String, Any>) {
-        let ottAnalyticsConfig = AnalyticsConfig(params:config)
+        let baseUrl: String = config["baseUrl"] as? String ?? ""
+        
+        let timerInterval: Double = config["timerInterval"] as? Double ?? 0.0
```

Comment:
What is default in plugin? What do we pass from config? I think 30sec is default

Reply:
yes
changed timerInterval => 30
and experimentalLiveMediaHit => true